server/controllers/comments.js

```diff
@@ -22,7 +111,13 @@ class CommentsController {
   static createComment(req, res, next) {
     const slug = req.params.article_slug;
     const { userId } = req;
-    const { comment } = req.body;
+    const {
```

Comment:
Good work but you can have lines 112 and 113 on one line using object destructuring. 
`const { userId, body: { comment } } = req;`